<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormWholeTicketSells</class>
 <widget class="QWidget" name="FormWholeTicketSells">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Date de Rapport</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>De:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEditStart"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>a:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEditEnd"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>dateEditEnd</zorder>
     <zorder>label_2</zorder>
     <zorder>dateEditStart</zorder>
     <zorder>label</zorder>
     <zorder></zorder>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPrintWholeSells">
       <property name="text">
        <string>Imprimer</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/img/img/printer.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
